django_netjsonconfig/base/template.py

```diff
@@ -53,6 +64,37 @@ class AbstractTemplate(BaseConfig):
                                                 'be automatically managed behind the scenes '
                                                 'for each configuration using this template, '
                                                 'valid only for the VPN type'))
+    sharing = models.CharField(_('Sharing'),
+                               choices=SHARING_CHOICES,
+                               default='private',
+                               max_length=16,
+                               db_index=True,
+                               help_text=_('Whether to keep this template private, share it publicly, '
+                                           'share it privately using a secret key or import its '
+                                           'contents from another source'))
+    key = models.CharField(max_length=64,
+                           unique=True,
+                           default=get_random_key,
+                           validators=[key_validator],
+                           help_text=_('share template key'))
+    description = models.TextField(_('Description'),
+                                   blank=True,
+                                   null=True,
+                                   help_text=_('Enter public description of this template'))
+    notes = models.TextField(_('Notes'),
+                             blank=True,
+                             null=True,
+                             help_text=_('Enter internal notes for the administrators'))
+    samples = JSONField(_('Sample Variables'),
+                        default=dict,
+                        blank=True,
+                        help_text=_('Enter some default values for the variables used by this template'),
+                        load_kwargs={'object_pairs_hook': OrderedDict},
+                        dump_kwargs={'indent': 4})
+    url = models.URLField(_('Import From'),
```

Comment:
can you please change `'Import From'` to `'Import URL'` and add a `help_text` which says `URL from which the template will be imported from`?